daemon/containerd/image_delete.go

```diff
@@ -35,6 +41,24 @@ import "github.com/docker/docker/api/types"
 // If prune is true, ancestor images will each attempt to be deleted quietly,
 // meaning any delete conflicts will cause the image to not be deleted and the
 // conflict will not be reported.
-func (i *ImageService) ImageDelete(imageRef string, force, prune bool) ([]types.ImageDeleteResponseItem, error) {
-	panic("not implemented")
+func (i *ImageService) ImageDelete(ctx context.Context, imageRef string, force, prune bool) ([]types.ImageDeleteResponseItem, error) {
+	// FIXME(thaJeztah): implement ImageDelete "force"
+	// FIXME(thaJeztah): implement ImageDelete "prune"
```

Comment:
Let me create tracking issues for these

Reply:
Created:

- https://github.com/moby/moby/issues/43849
- https://github.com/moby/moby/issues/43850